client/app/src/accounts/account.controller.js

```diff
@@ -1523,6 +1528,93 @@
         }
       };
 
+      function upload() {
+        var options = {
+          title: "Upload Image",
+          filters: [
+            { name: 'Images', extensions: ['jpg', 'png', 'gif'] }
+          ],
+          properties: ["openFile"],
```

Comment:
Unnecessary comma here

Reply:
I personally don't see a problem with this. It's valid syntax [see here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas). Also, it means it's 1 less line on a diff when an additional value is added at the end of the object.

Reply:
It's valid syntax, however, the linter which we are moving to (StandardJS) flags it by default. See **Trailing commas not allowed** here: https://standardjs.com/rules.html#javascript-standard-style

That being said, I do kind of like how it keeps the diff a bit cleaner.

Reply:
The syntax is valid, however, the linter which we are trying to move to (StandardJS) flags it by default. See **Trailing commas not allowed.** here: https://standardjs.com/rules.html#javascript-standard-style

That being said, I do kind of like the idea of keeping the diff a bit cleaner.

Reply:
The syntax is valid, however, the linter which we are trying to move to (StandardJS) flags it by default. See **Trailing commas not allowed.** here: https://standardjs.com/rules.html#javascript-standard-style

That being said, I do kind of like the idea of keeping the diff a bit cleaner.

Reply:
I'm all for a standard, so in that case I agree 😄 

I've always done trailing commas on arrays to save on the diffs. I can't argue with standards though xD

Reply:
I went ahead and removed the comma. I wasn't aware of the StandardJS style (and honestly I put it there on accident lol). Thanks guys!